HippoMocks/hippomocks.h

```diff
@@ -85,8 +85,15 @@ extern "C" __declspec(dllimport) void WINCALL DebugBreak();
 #define FUNCTION_STRIDE 1
 #endif
 
+#if defined (__SUNPRO_CC)
+#define SUNPRO_CC_VTABLE_INDEX 2
+#else
+#define SUNPRO_CC_VTABLE_INDEX 0
+#endif
+
 #if defined(_M_IX86) || defined(__i386__) || defined(i386) || defined(_X86_) || defined(__THW_INTEL) ||  defined(__x86_64__) || defined(_M_X64)
 #define SOME_X86
+#elif defined (sparc)
```

Comment:
It's an artifact. We were planning a version for the sun machines too. But then stopped, since the need was not worth it.